### PR TITLE
Feature/milestone4

### PR DESCRIPTION
**Pull request by trox115**
Filters categories working.

**Note to TSE:** I still have not the milestone 3 approved because it has not been reviewed by the time I made this PR.

### CODE REVIEW

File: src/components/FilterCategory.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  const filters = [];
+  const categories = [
+    'All',
+    'Action',
+    'Biography',
+    'Horror',
+    'History',
+    'Kids',
+    'Learning',
+    'Sci-fi',
```
Comment (bellom) [inline src/components/FilterCategory.js]: [Optional]
`first category`
we could practice `DRY` code means not repeating ourselves.
`categories` and `category` seems to have the same values.
I think we would create a file called `src/category.js` then use where ever we wanna use it in our code.

File: src/containers/BooksForm.js [hunk 2]
```diff
@@ -1,27 +1,98 @@
...
+  render() {
+    const category = [
+      'Action',
+      'Biography',
+      'Horror',
+      'History',
+      'Kids',
+      'Learning',
+      'Sci-fi',
+    ];
```
Comment (bellom) [inline src/containers/BooksForm.js]: `second category`

**Review by bellom — APPROVED** [review]
## Hi @trox115 :clap: 

## Congratulation :+1:  
_Go ahead and merge to dev and don't forget to submit project completion form.
Good luck with your next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

File: src/components/FilterCategory.js (hunk 1 above)
Comment (trox115) [reply src/components/FilterCategory.js]: I will do that. Thank you

**Comment by trox115** [comment]
Done the optional. Thank you :-)

**Pull request merged**